cmake/cpack_settings.cmake

```diff
@@ -18,9 +18,15 @@ endif()
 
 message(STATUS "Debian package version: ${CPACK_DEBIAN_PACKAGE_VERSION}")
 
-set(CCF_DEB_BASE_DEPENDENCIES
-    "libuv1 (>= 1.34.2);libc++1-10;libc++abi1-10;openssl (>=1.1.1)"
-)
+if(COMPILE_TARGET STREQUAL "sgx")
+  set(CCF_DEB_BASE_DEPENDENCIES
+      "libuv1 (>= 1.34.2);libc++1-10;libc++abi1-10;openssl (>=1.1.1)"
+  )
+else()
+  set(CCF_DEB_BASE_DEPENDENCIES
+      "libuv1 (>= 1.34.2);libc++1-12;libc++abi1-12;openssl (>=1.1.1)"
+  )
+endif()
```

Comment:
Should this be folded into the existing logic on line `33`, and `CCF_DEB_BASE_DEPENDENCIES` becomes `libuv1 (>= 1.34.2);openssl (>=1.1.1)`.